docs/index.rst

```diff
@@ -57,6 +54,10 @@ Installation
         :code:`pip install litestar[sqlalchemy]`
 
     :doc:`CLI </usage/cli>`
+        DEPRECATED (to be removed in v3) - Base installation now includes the CLI dependencies
+        and this group is no longer required to use the CLI. If you need the optional CLI dependencies,
+        install the ``standard`` group instead.
+
```

Comment:
Can we use the `.. deprecated::` [directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) here? I think it would look better.
```suggestion
        
```
```suggestion
        .. deprecated:: 3.0.0
        
        The `litestar` base installation now includes the CLI dependencies and this group is no longer required 
        to use the CLI. 
        If you need the optional CLI dependencies,install the ``standard`` group instead.

```